recipes/wolfssl/all/conanfile.py

```diff
@@ -156,4 +156,6 @@ def package_info(self):
             if self.settings.os in ["Linux", "FreeBSD"]:
                 self.cpp_info.system_libs.extend(["m", "pthread"])
             elif self.settings.os == "Windows":
-                self.cpp_info.system_libs.extend(["advapi32", "ws2_32"])
+                self.cpp_info.system_libs.extend(["advapi32", "ws2_32", "crypt32"])
```

Comment:
Is this line a bugfix? Was it needed for previous versions?

Reply:
Thanks for reviewing this PR. I just double checked and found that `crypt32` was not needed in previous versions. Also, `CoreFoundation` and `Security` frameworks are not needed in previous versions.